src/components/shared/Modal.css

```diff
@@ -1,19 +1,23 @@
 .modal-wrapper {
   position: fixed;
   display: flex;
+  flex-direction: column;
   align-items: center;
+  justify-content: flex-start;
   width: 100%;
   height: 100%;
   top: 0;
   left: 0;
+  padding-top: 30px;
```

Comment:
this will look a bit weird when in horizontal mode. might also justify a media query. it might also make sense to use this property on the modal itself and set the translate to 30px or 0px, since we have that positioning there anyway.